public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+    },
+    "chooseLandmarks": {
+      "title": "Configurer les Repères",
+      "body": "En groupe, décidez avec combien de points de repère vous voulez jouer et lesquels inclure dans le pool de sélection.",
+      "body_mapLandmark": "En groupe, décidez avec combien de points de repère vous voulez jouer <i>(en plus du repère de carte déjà inclus)</i>, et lesquels inclure dans le pool de sélection."
+    },
+    "chooseHirelings": {
+      "title": "Configurer les Mercenaires",
+      "body": "En groupe, décidez si vous voulez jouer avec des mercenaires et lesquels inclure dans le pool de sélection."
+    },
+    "postHirelingSetup": {
+      "subtitle": "Placer des Marqueurs de Mercenaires",
+      "body": "Placez les trois marqueurs mercenaires - marqués \"4\", \"8\", et \"12\" - sur les espaces \"4\", \"8\", et \"12\" de la piste de score sur la carte."
+    },
+    "drawCards": {
+      "title": "Tirez Cinq Cartes",
+      "body": "Chaque joueur pioche cinq cartes du paquet partagé. <i>(Vous choisirez trois cartes à conserver plus tard.)</i>"
+    },
+    "chooseFactions": {
+      "title": "Configurer les Factions",
+      "body": "En tant que groupe, sélectionnez les factions que vous souhaitez inclure dans le brouillon de faction."
+    },
+    "selectFaction": {
+      "subtitle": "Choisir la Faction",
+      "body": "Joueur {{count}}, choisissez la faction que vous souhaitez jouer dans le groupe de factions ci-dessous. Vous pouvez librement sélectionner des factions dans la liste pour voir leurs statistiques, avant de confirmer votre sélection en appuyant sur\"Etape Suivante.\""
+    },
+    "placeScoreMarkers": {
+      "title": "Placer les Marqueurs de Score",
+      "title_vagabondSetUp": "Mettez le Score et Marqueurs de relation",
+      "body": "Chaque joueur place le marqueur de score de la faction de son choix sur la case\"0\" de la piste de score.",
+      "body_vagabondSetUp": "Chaque joueur place le marqueur de score de la faction de son choix sur la case\"0\" de la piste de score. Chaque joueur Vagabond place un marqueur Relation pour chaque faction non Vagabonde en jeu sur la case Indifférent des Relations."
+    },
+    "chooseHand": {
+      "title": "Choisissez les Mains de Départ",
+      "body": "Chaque joueur choisit trois cartes dans sa main à conserver et place les deux autres cartes face cachée sur la pioche commune. Lorsque vous avez terminé, mélangez le jeu partagé."
+    },
+    "setupEnd": {
+      "title": "Commencer à jouer",
+      "body": "La configuration est maintenant terminée. Le jeu commencera avec le Joueur {{count}}. Si vous souhaitez redémarrer le processus de configuration, utilisez le bouton ci-dessous.<p><i>\"Root Automated Setup\" Développé par Ewen Cameron</i><br/><i>Basé sur le jeu de société\"Root\" publié par Leder Games</i><br/><i>Application sous licence CC BY-NC-SA 4.0</i></p>"
```

Comment:
Feel free to add an extra line below my credit to credit yourself for the translation 🙂

e.g. ...`Developed By Ewen Cameron</i><br/><i>French translation by RomainStorai</i><br/><i>Based on the board game`...